clustered/integration-test/src/test/java/org/ehcache/clustered/management/ClusteringManagementServiceTest.java

```diff
@@ -395,6 +399,60 @@ public static void initDescriptors() throws ClassNotFoundException {
     CACHE_DESCRIPTORS.add(new StatisticDescriptor("Cache:ClearCount", StatisticType.COUNTER_HISTORY));
     CACHE_DESCRIPTORS.add(new StatisticDescriptor("Cache:MissCount", StatisticType.COUNTER_HISTORY));
     CACHE_DESCRIPTORS.add(new StatisticDescriptor("Cache:MissRatio", StatisticType.RATIO_HISTORY));
+    //TODO CACHE_DESCRIPTORS above will be deletetd once the following task is completed: https://github.com/ehcache/ehcache3/issues/1684
+    /////////////////////////////////////////////////////////////////////////////////////////////////////
+
+    /////////////////////////////////////////////////////////////////////////////////////////////////////
+    //These descriptors are for the new statistis architecture.  These will remain.
+    //task https://github.com/ehcache/ehcache3/issues/1684
+    CACHE_DESCRIPTORS.add(new StatisticDescriptor("Cache:Get:HitNoLoaderCount", StatisticType.COUNTER_HISTORY));
+    CACHE_DESCRIPTORS.add(new StatisticDescriptor("Cache:Get:HitWithLoaderCount", StatisticType.COUNTER_HISTORY));
```

Comment:
Spoken to @anthonydahanne , I updated the issue (https://github.com/ehcache/ehcache3/issues/1684) to match the new requirements :-)
So we'll have to update this PR.